templates/email/subscription-frequency-changed.hypr

```diff
@@ -0,0 +1,16 @@
+{% extends "email/email" %}
+{% block body-content %}
+
+<p>{{ labels.subscriptionActivatedBlob|string_format(siteContext.generalSettings.websiteName, domainName)|safe}}</p>
```

Comment:
Even though the welcome blob for subscription activated and frequencyChanged is same, we should create a separate one for frequencyChanged.

Reply:
created a seperate label.